### ISSUE TITLE
Crash when updating postgresql with json data type

### ISSUE THREAD

**Issue by paragasu** [dresende/node-sql-query#43]
One of my postgresql table column is of type json;

```
{
   id: 4350,
   name: "hello world",
   setting: { notification: 'false', language: 'malay'}
 }
```

Passing the above value to the code

```
John.save(data, function(err){
})
```

Will result in crash 

```
//console.log('value', val, typeof val)
val value 4350 string
val value { notification: 'false', language: 'malay' } object
/home/paragasu/abby/dashboard/node_modules/sql-query/lib/Dialects/postgresql.js:67
    return "'" + val.replace(/\'/g, "''") + "'";
                     ^

TypeError: val.replace is not a function
```

**Comment by dxg** [comment]
I don't think JSON types are supported.

**Comment by cjnqt** [comment]
I needed this badly (in node-orm2) so I implemented support for json types here #44
